lib/rules/named-functions-in-promises.js

```diff
@@ -42,5 +47,11 @@ module.exports = {
         utils.isIdentifier(callee.property) &&
         promisesMethods.indexOf(callee.property.name) > -1;
     }
+
+    function isConciseArrowFunctionWithCallExpression(node) {
```

Comment:
I'd move this function to utilities, where other functions are, and write unit test for this specific function as well. What do you think?